README.md

```diff
@@ -80,7 +80,7 @@ npm start
 
 ## Author <a name="author"></a>
 
```

Comment:
- [ ] You have not implemented the task editing requirement. I am unable to edit a tasks description. Kindly include that functionality.

Reply:
- Kindly add the edit task functionality as previously suggested 👍🏽